infra-templates/k8s/34/rancher-compose.yml

```diff
@@ -67,6 +67,24 @@ catalog:
     required: false
     default: false
     type: boolean
+  - variable: SERVICE_CLUSTER_CIDR
+    label: Service Cluster IP CIDR
+    description: A CIDR notation IP range from which to assign service cluster IPs. This must not overlap with any IP ranges assigned to ipsec cidr (10.42.0.0/16).
+    required: true
+    default: "10.43.0.0/16"
+    type: string
+  - variable: DNS_CLUSTER_IP
+    label: KubeDNS cluster IP
+    description: The IP address assigned to kubedns service. This must be within the range of Service Cluster IP CIDR.
+    required: true
+    default: "10.43.0.10"
+    type: string
+  - variable: KUBEAPI_CLUSTER_IP
+    label: KubeAPI cluster IP for TLS certificate
+    description: This is the IP address that will be added as a SAN to the TLS certificate for kubernetes api, this must matches the IP of the kubernetes master service.
```

Comment:
Similar to dns, we should mention that the ip address must be the first ip of the Service Cluster IP CIDR

Reply:
got it, will add this part